IceWhaleTech/CasaOS-AppManagement
# feat: add OPENAI_API_KEY env var to v2app from config file

# What does the PR do?
It imports a new config and adds an env var(`OPENAI_API_KEY`) to every v2 app.

# Test Steps
1. change the config(`/etc/casaos/app-management.conf`)
2. new an app
3. type `env` in app terminal like
![image](https://github.com/IceWhaleTech/CasaOS-AppManagement/assets/29306285/a68e3053-d6bf-4fb7-bc12-bb5f7b4a65e0)

4.  use `go run . app-management set global OPENAPI_AI_KEY sk-test-value` in https://github.com/IceWhaleTech/CasaOS-CLI/pull/34


5. type `env` in app terminal like. to see `OPENAI_API_KEY` be changed.
